homeassistant/components/august/binary_sensor.py

```diff
@@ -261,7 +276,7 @@ class AugustDoorbellBinarySensor(AugustEntityMixin, BinarySensorEntity):
     def __init__(
```

Comment:
All the doorbells should move to event entities in the future after a 6 month breaking change window.  I need to get the user who has the older doorbells to help out to make that happen though to avoid breaking the old doorbells and they aren't available until later in the year.